core/domain/suggestion_services.py

```diff
@@ -1826,3 +2020,866 @@ def update_question_suggestion(
+        suggestion_models.TranslationContributionStatsModel.create(
+            language_code=suggestion.change.language_code,
+            contributor_user_id=suggestion.author_id,
+            topic_id=topic_id,
+            submitted_translations_count=1,
+            submitted_translation_word_count=content_word_count,
+            accepted_translations_count=0,
+            accepted_translations_without_reviewer_edits_count=0,
+            accepted_translation_word_count=0,
+            rejected_translations_count=0,
+            rejected_translation_word_count=0,
+            contribution_dates=[_get_date_as_string(suggestion.last_updated)]
+        )
+    else:
+        translation_contribution_stat = (
+            create_translation_contribution_stats_from_model(
+                translation_contribution_stat_model))
+
+        suggestion_is_accepted = (
+            suggestion.status == suggestion_models.STATUS_ACCEPTED
+        )
+
+        increment_translation_contribution_stats_at_review(
+            translation_contribution_stat, content_word_count,
+            suggestion_is_accepted, suggestion.edited_by_reviewer)
+        _update_translation_contribution_stats_models(
+            [translation_contribution_stat])
+
+
+def update_translation_review_stats(
+    suggestion: suggestion_registry.BaseSuggestion
+) -> None:
+    """Creates/updates TranslationReviewStatsModel model for given translation
+    reviewer when a translation is reviewed.
+
+    Args:
+        suggestion: Suggestion. The suggestion domain object that is being
+            reviewed.
+    """
+    topic_id = None
+    exp_opportunity = (
+        opportunity_services.get_exploration_opportunity_summary_by_id(
+            suggestion.target_id))
+    # We can confirm that exp_opportunity will not be None since there should
+    # be an assigned opportunity for a given translation. Hence we can rule out
+    # the possibility of None for mypy type checking.
+    assert exp_opportunity is not None
+    topic_id = exp_opportunity.topic_id
+    suggestion_is_accepted = (
+        suggestion.status == suggestion_models.STATUS_ACCEPTED
+    )
+
+    content_plain_text = html_cleaner.strip_html_tags(
+        suggestion.change.translation_html)
+    content_word_count = len(content_plain_text.split())
+
+    translation_review_stat_model = (
+        suggestion_models.TranslationReviewStatsModel.get(
+            suggestion.change.language_code, str(suggestion.final_reviewer_id),
+            topic_id
+        ))
+
+    if translation_review_stat_model is None:
+        suggestion_models.TranslationReviewStatsModel.create(
+            language_code=suggestion.change.language_code,
+            reviewer_user_id=str(suggestion.final_reviewer_id),
+            topic_id=topic_id,
+            reviewed_translations_count=1,
+            reviewed_translation_word_count=content_word_count,
+            accepted_translations_count=(1 if suggestion_is_accepted else 0),
```

Comment:
Hm, see https://github.com/oppia/oppia/pull/15917#discussion_r961954757 -- we're trying to do this semantically correctly and not change types of things. That's because technically suggestion_is_accepted doesn't actually represent an int in the first place.

If there are further issues below we should fix them similarly as here, I think.

Reply:
Made everything as `1 if <boolean> else 0`